auth/src/main/java/com/sangyunpark/auth/application/AuthService.java

```diff
@@ -33,11 +32,10 @@ public TokenResponseDto login(final LoginRequestDto loginRequestDto) {
         return new TokenResponseDto(token);
     }
 
-    public TokenResponseDto reissue(final String refreshToken, final UserPrincipal userPrincipal) {
-        final String email = userPrincipal.getEmail();
+    public TokenResponseDto reissue(final String refreshToken, final String email, final String userType, final String userStatus) {
         tokenProvider.validateToken(refreshToken);
         validateStoredRefreshToken(email, refreshToken);
-        Token newToken = generateAndStoreToken(email, userPrincipal.getUserType(), userPrincipal.getUserStatus());
+        Token newToken = generateAndStoreToken(email, UserType.valueOf(userType), UserStatus.valueOf(userStatus));
         return new TokenResponseDto(newToken);
```

Comment:
return new TokenResponseDto(this.generateAndStoreToken(email, UserType.valueOf(userType), UserStatus.valueOf(userStatus)));